autogluon/task/tabular_prediction/predictor.py

```diff
@@ -31,13 +31,13 @@ class TabularPredictor(BasePredictor):
             What metric is used to evaluate predictive performance.
         label_column : str
             Name of table column that contains data from the variable to predict (often referred to as: labels, response variable, target variable, dependent variable, Y, etc).
-        feature_types : :class:`autogluon.utils.tabular.features.feature_types_metadata.FeatureTypesMetadata`
+        feature_metadata : :class:`autogluon.utils.tabular.features.feature_metadata.FeatureMetadata`
             Inferred data type of each predictive variable after preprocessing transformation (i.e. column of training data table used to predict `label_column`).
-            Contains both raw dtype and special dtype information. Each feature has exactly 1 raw dtype (such as 'int', 'float', 'category') and zero to many special dtypes (such as 'datetime', 'text', 'text_ngram').
+            Contains both raw dtype and special dtype information. Each feature has exactly 1 raw dtype (such as 'int', 'float', 'category') and zero to many special dtypes (such as 'datetime_as_int', 'text', 'text_ngram').
             Special dtypes are AutoGluon specific feature types that are used to identify features with meaning beyond what the raw dtype can convey.
-                `feature_types.get_feature_types_raw_flattened()`: Dictionary of feature name -> raw dtype mappings.
-                `feature_types.feature_types_raw`: Dictionary of lists of raw feature names, grouped by raw feature dtype.
-                `feature_types.feature_types_special`: Dictionary of lists of special feature names, grouped by special feature dtype.
+                `feature_metadata.type_map_raw`: Dictionary of feature name -> raw dtype mappings.
+                `feature_metadata.type_group_map_raw`: Dictionary of lists of raw feature names, grouped by raw feature dtype.
+                `feature_metadata.type_group_map_special`: Dictionary of lists of special feature names, grouped by special feature dtype.
```

Comment:
For future PR can we make it so `predictor.feature_metadata` or at least `print(predictor.feature_metadata)`prints out these 3 items?  I think most users just want to view this information once, but it's not easy to remember these long names

Reply:
added print(predictor.feature_metadata)